app/views/courses/_entry_requirements.html.erb

```diff
@@ -0,0 +1,12 @@
+<p class="govuk-body">To be a teacher in England you need:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>a degree or an equivalent qualification</li>
+  <li>English and Maths GCSEs at grade 4 (C) or above</li>
+  <li>For primary teaching, a science subject GCSE at grade 4 (C) or above</li>
+</ul>
+<p class="govuk-body">You can find out more about entry requirements, and how it works if you have studied overseas, at <%= link_to "Get Into Teaching", "https://beta-getintoteaching.education.gov.uk/steps-to-become-a-teacher" %>.</p>
```

Comment:
this needs be gov_uk_link_to please 🙏

Reply:
I've done so for both, previously the link style was being inherited from the class on the <p> tag. Is this a govuk standard for screen readers or something? Just asking so I know for next time 🙃

Reply:
`govuk_link_to` is an extension of the Rails link helper which adds the `govuk-link` class to the link. You need to add this class to links so they use the right colour (a CSS architecture decision which causes no end of problems, but that’s a whole other rant 😅)